### ISSUE TITLE
Bug: Imported sensor update policy error due to `schedule.timezone` attribute

### ISSUE THREAD

**Issue by stmyers** [CrowdStrike/terraform-provider-crowdstrike#32]
Hi, me again 😄 

I've been working on importing our existing CrowdStrike config into state and ran into an issue while importing a `crowdstrike_sensor_update_policy`.  It seems when importing and schedule is disabled, the imported state file ends up with 
```terraform
schedule  = {
        enabled  = false
        timezone = ""
    }
```

The `timezone = ""` attribute ending up in state causes errors in subsequent runs because TF tries to set the timezone to default:
```terraform
~ schedule = {
          ~ timezone = "" -> "Etc/UTC"
            # (1 unchanged attribute hidden)
        }
```

but if `schedule` is disabled, `timezone` shouldn't be provided:
```terraform
 Error: Invalid schedule block: timezone provided
 │ To implement idempotency timezone and time_blocks should not be provided when enabled is false.
 | Operation failed: failed running terraform plan (exit 1)
```

I currently work-around this with a `lifecycle.ignore_changes` block, and I'm able to proceed.

So here's my current working block, which ideally I'd like to remove the `lifecycle.ignore_changes`:

```terraform
resource "crowdstrike_sensor_update_policy" "win_n_minus_1" {
  name                    = "Windows Sensor Update N-1"
  platform_name           = "Windows"
  description             = "Windows Sensor Update Policy for N-1 deferment."
  enabled                 = true
  build                   = data.crowdstrike_sensor_update_policy_builds.builds.windows.n1.build
  uninstall_protection    = true
  schedule                = {enabled = false}
  host_groups             = [
                              crowdstrike_host_group.servers.id,
                              crowdstrike_host_group.workstations.id,
                            ]
  
  
  # Ignore changes to the timezone to avoid bug - will open an issue in Github to address
  #  | Error: Invalid schedule block: timezone provided
  #  │ To implement idempotency timezone and time_blocks should not be provided when enabled is false.
  #  | Operation failed: failed running terraform plan (exit 1)
  lifecycle {ignore_changes = [schedule.timezone]}
}
```

**Comment by ffalor** [comment]
@stmyers Sorry this fell through the cracks on my part!

This error is custom validation. It looks like the import defaults timezone to an empty string. Which causes the `terraform state show` command to fill this property out.

removing the `timezone = ""` line and running terraform apply should work without requiring you to add `lifecycle {ignore_changes = [schedule.timezone]}`

I'll see if I can prevent `terraform import` from setting the `timezone` unless the shedule is actually enabled to prevent you from having to remove the `timezone` property when doing imports.

**Comment by ffalor** [comment]
Fixed in version `v0.0.7`

```terraform
terraform {
  required_providers {
    crowdstrike = {
      source = "CrowdStrike/crowdstrike"
      version = "0.0.7"
    }
  }
}
```
